### PR TITLE
Awesome Books Development

### PR DESCRIPTION
**Pull request by stephnna**
Hi Code Reviewer,

my code partner and I implemented the awesome book.
First we set up linters, created HTML, CSS and js files and implemented the code.

Kindly review for your approval

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  books.push(newBook);
+  displayBook();
+  localStorage.setItem('allbooks', JSON.stringify(books));
+  deleteBook();
+});
+
+// filter out deleted book
+const filtered = books.filter((book) => book !== null);
+books = [...filtered];
+localStorage.setItem('allbooks', JSON.stringify(books));
```
Comment (elyor-doniyorov) [inline script.js:52]: - [ ] Currently, when page reload after adding a book and removing a book, data is not preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md). Kindly fix that issue and request for another review.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @stephnna and @oluwajuwon8, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
✅ There are no linter errors
✅ There are add/remove functionalities


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (stephnna) [reply script.js]: Thanks!
fixed

Comment (stephnna) [reply script.js]: Thanks!
fixed

Comment (stephnna) [reply script.js]: Thanks!
fixed

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @stephnna, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**